### PR TITLE
New psic_classification

### LINKED ISSUE #199: Sector bridge - PSIC
Hello all!

We have another sector classification bridge to include in the r2dii.data package. This time this is the Philippine Standard Industrial Classification (PSIC), used by one of the banks that has just started with the implementation of PACTA for banks.

Please find attached the file with the corresponding info. Let me know if there is any question or comment.

Thanks a lot!
Daisy

@georgeharris2deg 
@Lauramirez-2ii 
[NSCB_PACTA_bridge.xlsx](https://github.com/2DegreesInvesting/r2dii.data/files/5913348/NSCB_PACTA_bridge.xlsx)

### PR DESCRIPTION
**Pull request by maurolepore**
Closes #199

Thanks @daisy-pachecho

(Created following the new helpers and article use_classification in r2dii.usethis.)

### CODE REVIEW

File: data-raw/data_dictionary/psic_classification.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+dataset,column,typeof,definition
+psic_classification,original_code,character,Original psic_classification sector name
+psic_classification,code_level,double,Level of granularity of psic_classification code
+psic_classification,code,double,Formatted psic_classification code
+psic_classification,sector,character,Associated 2dii sector
+psic_classification,borderline,logical,Flag indicating if 2dii sector and classification code are a borderline match. The value TRUE indicates that the match is uncertain between the 2dii sector and the classification. The value FALSE indicates that the match is certainly perfect or the classification is certainly out of 2dii's scope.
```
Comment (maurolepore) [inline data-raw/data_dictionary/psic_classification.csv:6]: TODO: Here I need to check that the colums indeed match the columns that Daisy provided.

File: tests/testthat/test-data_dictionary.R [hunk 2]
```diff
@@ -31,6 +31,7 @@ test_that("defines the expected objects", {
     "sector_classifications",
     "sic_classification"
   )
+  expected_datasets <- c(expected_datasets, 'psic_classification')
```
Comment (maurolepore) [inline tests/testthat/test-data_dictionary.R:34]: Here is an alternative way to extend `expected_datasets`. The old way is nicer but this one is easier to program.

Comment (maurolepore) [reply tests/testthat/test-data_dictionary.R:34]: TODO: Do I need to add a new step in r2dii.usethis to add a snapshot for the new dataset?

**Review by jdhoffa — APPROVED** [review]
This looks good to me, thanks for taking care of this so quickly @maurolepore and thanks for flagging @daisy-pacheco :-)

File: tests/testthat/test-data_dictionary.R (hunk 2 above)
Comment (maurolepore) [reply tests/testthat/test-data_dictionary.R:34]: I now think I'm not missing a step; it think the snapshot of `sector_classification` accounts for it.

**Comment by daisy-pacheco** [comment]
Thank you both @maurolepore and @jdhoffa for this! I just have one follow-up question: What would be the best way for me to know whether this has been pushed to CRAN or not? Should I just give it a try on my own pc by reinstalling the package, or is there another way to make sure that for the following session with our partner they can already have the possibility to use the corresponding bridge? Thanks again! :)

**Comment by maurolepore** [comment]
For 2DII people like you, the easiest way might be to ask us on slack. For external people it may be to watch the News section of the package website.

[Here](https://2degreesinvesting.github.io/r2dii.data/news/index.html) it is.

**Comment by maurolepore** [comment]
And the answer if: No, it's still not on CRAN but I can make it happen as fast as in a few hours. Do you need it soon? @jdhoffa, is there anything else you would like to add to the next release?

**Comment by daisy-pacheco** [comment]
Thanks for the quick answer @maurolepore :) next time I'll make it through Slack. Just to follow the conversation here, it would be great if we could have it on CRAN for the next session with our partner, which is tomorrow :) but let me know in any case and we could tell them it would be ready for next session. Thanks again :)

**Comment by maurolepore** [comment]
Great! I'll start now. BTW, asking here is great too -- maybe even better because it is tightly connected with the "feature" you are interested in seeing published.
:smile:

**Comment by maurolepore** [comment]
FYI, this is my WIP: https://github.com/2DegreesInvesting/r2dii.data/issues/208

**Pull request merged**